### ISSUE TITLE
Question 5 test expects different answer from prompt

### ISSUE THREAD

**Issue by RDegnen** [ga-wdi-boston/ruby-objects-diagnostic#74]
``` rb
# Question 5
# Define a new class, 'Lion', which (a) inherits from 'Cat',
# (b) uses the 'Carnivorous' module below as a mixin, and
# (c) adds a new method called `roar`, which prints out "ROAR!"
```

The test expects `"ROAR"` instead of `"ROAR!"` which has a `!`

**Comment by jrhorn424** [comment]
Addressed in 8632d4f02b4a9717fee323f24682049835b97594

**Comment by RealWeeks** [comment]
@jrhorn424 I'll answer for @RDegnen on this one. Looks good.
